test/Fencer/Server/Test.hs

```diff
@@ -85,6 +87,49 @@ test_serverResponseNoRules =
       , Proto.rateLimitRequestHitsAddend = 0
       }
 
+-- | Test that when Fencer is started and there are no rules after
+-- loading (i.e., 'reloadRules' has been ran, but there is no
+-- configuration), requests to Fencer will be responded to with OK.
+--
+-- This behavior matches @lyft/ratelimit@.
+test_serverResponseNoRules :: TestTree
+test_serverResponseNoRules =
+  withResource createServer destroyServer $ \serverIO ->
+    testCase "When no rules have been loaded, all responses are OK" $ do
+      server <- serverIO
+      atomically (setRules (serverAppState server) []) -- an empty rule list
+      withService server $ \service -> do
+        response <- Proto.rateLimitServiceShouldRateLimit service $
+          Grpc.ClientNormalRequest request 1 mempty
+        expectSuccess
+          (expectedResponse, Grpc.StatusOk)
+          response
+  where
+    request :: Proto.RateLimitRequest
+    request = Proto.RateLimitRequest
+      { Proto.rateLimitRequestDomain = "domain"
+      , Proto.rateLimitRequestDescriptors =
+          fromList $
+          [ Proto.RateLimitDescriptor $
+              fromList [Proto.RateLimitDescriptor_Entry "key" "value"]
+          ]
+      , Proto.rateLimitRequestHitsAddend = 0
+      }
+
+    expectedResponse :: Proto.RateLimitResponse
+    expectedResponse = Proto.RateLimitResponse
```

Comment:
This is `genericOKResponse` from #107, right?

Reply:
Well spotted! I'm copying it here too in an upcoming commit.